lib/ethpy/ethpy/hyperdrive/api/api.py

```diff
@@ -777,11 +853,11 @@ def calc_bonds_given_shares_and_rate(
         r"""Returns the bond reserves for the market share reserves
         and a given fixed rate.
 
-
-        The calculation is based on the formula:
+        The function does not perform contract calls, but instead relies on the Hyperdrive-rust sdk
+        to simulate the contract outputs. The calculation is based on the formula:
 
         .. math::
-            mu * (z - zeta) * (1 + apr * t) ** (1 / tau)
+            \mu * (z - \zeta) * (1 + \text{apr} * t)^{1 / \tau}
```

Comment:
we don't define tau here, but neither does the rust sdk. mental note to attempt to remove all mentions of tau in a future cleanup pass-through.

Reply:
yeah agreed -- we should make this consistent across sol -> rust -> python